lib/Service/src/Util.h

```diff
@@ -85,35 +85,53 @@ namespace Util
 
     /**
      * Divide and round.
-     * 
+     *
      * @param[in] numerator     The numerator.
      * @param[in] denominator   The denominator.
-     * 
+     *
      * @return Result
      */
     uint32_t divRoundUp(uint32_t numerator, uint32_t denominator);
 
     /**
      * Divide and round.
-     * 
+     *
      * @param[in] numerator     The numerator.
      * @param[in] denominator   The denominator.
-     * 
+     *
      * @return Result
      */
     int32_t divRoundUp(int32_t numerator, int32_t denominator);
 
     /**
      * Is button triggered?
      * Triggered means a pressed/released change.
-     * 
+     *
      * @param[in]       button      The button.
      * @param[inout]    lastState   The last button state.
-     * 
+     *
      * @return If button is triggered, it will return true otherwise false.
      */
     bool isButtonTriggered(IButton& button, bool& lastState);
 
+    /**
+     * Convert a speed in encoder steps per second to a speed in mm/s.
+     *
+     * @param[in]   speedStepsPerSec Speed in encoder steps per second
+     *
+     * @return Speed in mm/s
+     */
+    int32_t stepsPerSecondToMillimetersPerSecond(int16_t speedStepsPerSec);
+
+    /**
+     * Convert a speed in mm/s to a speed in encoder steps per second.
+     *
+     * @param[in]   speedMmPerSec Speed in mm/s
+     *
+     * @return Speed in encoder steps per second
+     */
+    int16_t millimetersPerSecondToStepsPerSecond(int32_t speedMmPerSec);
```

Comment:
Keep in mind that its no float, the accuracy will decrease! What is the smallest angle and the smallest length you can drive?

Reply:
Not sure about the smallest angle and lenght, but the smallest speed that can be driven is 8 steps/s using `RobotConstants::ENCODER_STEPS_PER_M = 8043U`

Reply:
1.35 °/s

If this is accurate enough, its ok.